### ISSUE TITLE
Smart contract governance

### ISSUE THREAD

**Issue by jandrad** [hivebp/verifystatus#1]
In order to create a fully decentralized contract, proper governance must be defined.

Key points to consider:

* How is an update deployed?
* How to add a new marketplace?
* Proper documentation of the smart contract for integrators.

**Comment by hiveben** [comment]
Yes, happy for any suggestions going forward. Documentation with sample code is in the works. It's just been a couple of stressful weeks

**Comment by jandrad** [comment]
@hiveben In my opinion the ownership of the contract should be delegated to a group of interested parties. Every deployment should have a multi-sig transaction that needs to be approved by at least 2 thirds of the parties.

**Comment by hiveben** [comment]
@jandrad agreed. We can add you and Atomichub might agree as well. Probably also the WAX team itself

**Comment by jandrad** [comment]
@hiveben it should be the parties actually interested in using the contract so we don't end up in a situation where things cannot move forward.

**Comment by hiveben** [comment]
@jandrad I think WAX is very interested for their own place. Atomichub probably not as much, but they agreed to support pushing verifications.

**Comment by jandrad** [comment]
@hiveben if there is support for that then it would work.
